test/cmd/admin.sh

```diff
@@ -155,6 +155,30 @@ os::cmd::expect_success 'oadm policy add-cluster-role-to-group cluster-admin sys
 os::cmd::expect_success 'oadm policy remove-cluster-role-from-group cluster-admin system:unauthenticated'
 os::cmd::expect_success 'oadm policy add-cluster-role-to-user cluster-admin system:no-user'
 os::cmd::expect_success 'oadm policy remove-cluster-role-from-user cluster-admin system:no-user'
+# test the rolebinding and clusterrolebinding name flag
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa1 -N rb1'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa2 -N rb2'
+os::cmd::expect_success 'oadm policy add-role-to-user admin -z fake-sa3 -N rb2'
+os::cmd::expect_success_and_text 'oc get rolebinding/rb1' 'rb1       /admin                        fake-sa1'
+os::cmd::expect_success_and_text 'oc get rolebinding/rb2' 'rb2       /admin                        fake-sa2, fake-sa3'
+os::cmd::expect_success 'oadm policy remove-role-from-user admin -z fake-sa3 -N rb2'
```

Comment:
Specifically -- a lot of the create and check flows seem superfluous -- we should be able to check that the struct of `Options` would lead to the correct objects being passed to the client in a unit test. Writing the test like this outside only also checks that the client correctly updates the server, which we don't need to be doing here.